beesdoo_pos/static/src/js/beesdoo.js

```diff
@@ -5,14 +5,11 @@ odoo.define("beesdoo_pos.screens", function (require) {
 
     models.load_fields("res.partner", "is_company");
 
-    var set_customer_info = function (el_class, value, prefix) {
-        var el = this.$(el_class);
-        el.empty();
-        if (prefix && value) {
-            value = prefix + value;
-        }
+    var set_customer_info = function (parent_class, value) {
+        var parent = this.$(parent_class);
+        var info_div = $("<div></div>").text(value);
```

Comment:
this seems like a strange way to create an element. i don’t know what the best way is, but here is another way:
```suggestion
        var info_div = document.createElement("div");
        info_div.textContent = value;
```
and then no need for the `[0]` below.

Reply:
Thx, I've read that the way I did this is the standard way with jquery ([see here ](https://www.w3schools.com/jquery/jquery_dom_add.asp)for instance), but your way is cleaner :)

Reply:
thanks for the information. the jquery way is ok for me (although there is no `[0]` in the example). in any case, a comment explaining what is being done would be nice to have, since it’s not obvious. something like: `create a new div containing value`.

Reply:
Sorry I was not clear, I used your way in the new fixup commit :)